runner/runner.go

```diff
@@ -114,6 +116,9 @@ func New(options *Options) (*Runner, error) {
 	httpxOptions.RetryMax = options.Retries
 	httpxOptions.FollowRedirects = options.FollowRedirects
 	httpxOptions.FollowHostRedirects = options.FollowHostRedirects
+	if options.Favicon {
+		httpxOptions.FollowRedirects = true
+	}
```

Comment:
We can remove this part, as enabling this option automatically will result in different results for probing when using with or without `-favicon` option.